var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -85,6 +86,14 @@ def setup_run_environment(self, env):
         env.set('CUDA_HOME', self.prefix)
 
     def install(self, spec, prefix):
+        if os.path.exists('/tmp/cuda-installer.log'):
+            try:
+                os.remove('/tmp/cuda-installer.log')
+            except Exception:
```

Comment:
```suggestion
            except OSError:
```
We don't want to catch all exceptions, just those that occur when trying to remove the file.